src/core/blockchain/celo/client-utils.ts

```diff
@@ -69,25 +68,26 @@ export class ClientUtils implements IClientUtils {
         const token = await this.getToken(txInfo.to);
         const data: any = {};
 
-        if (token.type === TokenType.ERC20) {
-            try {
-                const transferInputParameteres = this.decodeInputData(
-                    'transfer(address,uint256)',
-                    txInfo.input
-                );
-                if (transferInputParameteres[0] && transferInputParameteres[1]) {
-                    data.params = [
-                        '0x' + transferInputParameteres[0],
-                        transferInputParameteres[1].toString(10)
-                    ];
-                } else {
-                    throw new Error('Cannot decode input data');
-                }
-            } catch (e) {
-                // probably not a transaction
-                return null;
-            }
-        }
+        // TODO - find how to get the data since its a contract call not simple transfer call
```

Comment:
by commenting this I think ERC20 txs will not be showed ok in wallet.

Reply:
This would apply for inbound transactions

Reply:
#868 this will implement the entire functionality for staking transactions. I enabled the transactions for erc20